MyWeightTests/UITableViewExtensionsTests.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  UITableViewExtensionsTests.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import XCTest
+@testable import MyWeight
+
+class InvarianteTableViewCell: UITableViewCell { }
+class EmojiTableViewCell😎: UITableViewCell { }
+class a0_$WÅ´Ø: UITableViewCell { }
+
+class UITableViewExtensionsTests: XCTestCase {
+    private var tableView: UITableView!
+    
+    override func setUp() {
+        super.setUp()
+        tableView = UITableView()
+    }
+    
+    override func tearDown() {
+        tableView = nil
+        super.tearDown()
+    }
+    
+    func testDefaultReuseIdentifier() {
+        var cellClass = UITableViewCell.self
+        var cellDefaultReuseIdentifier = cellClass.defaultReuseIdentifier
+        var expectedDefaultReuseIdentifier = "UITableViewCell"
+        XCTAssertEqual(cellDefaultReuseIdentifier, expectedDefaultReuseIdentifier)
```

Comment:
I like also the idea of a app "100% third party free". Technically using Nimble will remain like this, because will be only on tests, hehe. But it is cheating ¯\\\_(ツ)\_/¯
I'm also interested in learn more about RxSwift.

What do you think about go 100% our code for now, and then we rethink about it in the future?